### ISSUE TITLE
Feature Request: Shutdown remote server after disconnect

### ISSUE THREAD

**Issue by t1u1** [microsoft/vscode-remote-release#10205]
After disconnecting from a remote, it would be good to shutdown remote servers automatically. These services consume several Gigs of RAM, so it isn't ideal to have them running around for long.

If there was a way to configure the shutdown period, that would be nice. Then people who need the servers lingering can keep a high timeout value. While those who would like to conserve memory on the remote can set a lower timeout.

**Comment by joshspicer** [comment]
Thanks for the idea.  This kind of action might be better performed by the user outside of Remote - SSH extension (I am assuming from context you're using that extension).  You didn't mention what platform you are on, but [here's an idea on linux on how you could accomplish what you're looking for](https://stackoverflow.com/a/57004045).

**Comment by t1u1** [comment]
Indeed, I am using Remote-SSH extension, and logging into a minimal but persistent Linux container.

Killing the remote server in an outside script has the problem that if a user logs out from a terminal session, it might kill a server that is still connected to VSCode.

**Comment by t1u1** [comment]
It looks like #440 achieves the same thing as this request (with a small timeout). If so, you can close this issue.
